RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexCA15Computer_cfi.py

```diff
@@ -6,9 +6,9 @@
     trackSelectionBlock,
     beta = cms.double(1.0),
     R0 = cms.double(1.5),
-    maxSVDeltaRToJet = cms.double(1.3),
+    maxSVDeltaRToJet = cms.double(1.0),
```

Comment:
It depends how you look at it. The new training was done with a reduced cut value, it also had to be changed in the release.

I updated the PR description.